e2e-tests/spring-examples/src/test/java/org/evomaster/e2etests/spring/examples/db/base/DbBaseEMTest.java

```diff
@@ -1,14 +1,14 @@
 package org.evomaster.e2etests.spring.examples.db.base;
 
-import org.evomaster.core.Main;
 import org.evomaster.core.problem.rest.HttpVerb;
 import org.evomaster.core.problem.rest.RestIndividual;
 import org.evomaster.core.search.Solution;
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-public class DbBaseEMTest extends DbBaseTestBase {
+public class
+DbBaseEMTest extends DbBaseTestBase {
 
```

Comment:
why on 2 lines?

Reply:
It appears to be a phantom ENTER keypress... Fixed.